observability/network_observability/configuring-operator.adoc

```diff
@@ -6,28 +6,29 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update the Flow Collector API resource to configure the Network Observability Operator and its managed components. The  Flow Collector is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`.
+You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference].
 
 
 include::modules/network-observability-flowcollector-view.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-For more information about conversation tracking, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-working-with-conversations_nw-observe-network-traffic[Working with conversations].
+* xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference]
+* xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-working-with-conversations_nw-observe-network-traffic[Working with conversations]
```

Comment:
Same comment about link text (although I recognize this is out of scope to your work, up to you):
```suggestion
* xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-working-with-conversations_nw-observe-network-traffic[Working with conversation tracking]
```